app/models/speedrun_dot_com_user.rb

```diff
@@ -0,0 +1,26 @@
+require 'speedrundotcom'
+
+class SpeedrunDotComUser < ApplicationRecord
+  belongs_to :user
+
+  def self.from_api_key!(api_key, user:)
+    srdc_user = SpeedrunDotComUser.find_by(api_key: api_key)
```

Comment:
If below on line 13 we're going to leave off the `SpeedrunDotComUser` for the `new` call, I feel like it should also be left off of the `find_by` call.